azblob/chunkwriting.go

```diff
@@ -28,7 +28,7 @@ type blockWriter interface {
 // well, 4 MiB or 8 MiB, and autoscale to as many goroutines within the memory limit. This gives a single dial to tweak and we can
 // choose a max value for the memory setting based on internal transfers within Azure (which will give us the maximum throughput model).
 // We can even provide a utility to dial this number in for customer networks to optimize their copies.
-func copyFromReader(ctx context.Context, from io.Reader, to blockWriter, o UploadStreamToBlockBlobOptions) (*BlockBlobCommitBlockListResponse, error) {
+func copyFromReader(ctx context.Context, from io.Reader, to blockWriter, o UploadStreamToBlockBlobOptions, cpk ClientProvidedKeyOptions) (*BlockBlobCommitBlockListResponse, error) {
```

Comment:
>cpk ClientProvidedKeyOptions [](start = 107, length = 28)

this is not getting used though?

Reply:
Oh that's embarrassing. I didn't notice. Rectified.